CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 ## Unreleased
 
+### Migration notes
+
+### 🌟 New javaagent instrumentation
+
+- Add instrumentation for vertx-sql-client
+  ([#8311](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8311))
+
+### 🌟 New library instrumentation
+
+### 📈 Enhancements
+
+- Make net.transport an optional attribute
+  ([#8279](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8279))
+- AddingSpanAttributes annotation
+  ([#7787](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7787))
+- JavaScript Snippet Injection
```

Comment:
Does it need title capitalization?

Reply:
I don't think so...but that was the original case of the PR title.
```suggestion
- JavaScript snippet Injection
```